src/Microsoft.ML.AutoML/AutoMLExperiment/AutoMLExperiment.cs

```diff
@@ -248,94 +248,93 @@ public async Task<TrialResult> RunAsync(CancellationToken ct = default)
                 var parameter = tuner.Propose(trialSettings);
                 trialSettings.Parameter = parameter;
 
-                using (var trialCancellationTokenSource = new CancellationTokenSource())
+                var trialCancellationTokenSource = new CancellationTokenSource();
```

Comment:
This is the only actual change: removing `using` statement and leave disposing of trialCancellationTokenSource to GC